### PR TITLE
[CBRD-23629] Fix slip of javasp utility

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CBRD-23629

This PR fixes the following regression. It caused because an error is not cleared before starting JVM in javasp_start_server (). errors created by er_set() in javasp_is_running () was detected by jsp_start_server() and was handled incorrectly. it should be cleared.

**Repro.**  
```
cubrid javasp start demodb
ps -ef | grep cub_javasp
kill -9 <pid>
cubrid javasp start demodb
```

**Expected**
```
@ cubrid javasp start: demodb
++ cubrid javasp start: success
```

**Actual** 
```
@ cubrid javasp start: demodb
Can't connect Java VM: connect()
++ cubrid javasp start: fail
```

And I fixed some trivial slip.

### CODE REVIEW

File: src/executables/javasp.cpp [hunk 1]
```diff
@@ -211,7 +211,7 @@ main (int argc, char *argv[])
 	    goto exit;
 	  }
 
-	status = javasp_start_server (jsp_info, db_name, pathname.c_str ());
+	status = javasp_start_server (jsp_info, db_name, pathname);
```
Comment (byungwook-kim) [inline src/executables/javasp.cpp:214]: what is different pathname.c_str() and pathname?

Comment (hgryoo) [reply src/executables/javasp.cpp:214]: c_str() returns char* of std::string. the type of an argument of javasp_start_server is std::string&.
it is just a slip that causes creating new std::string from char* by the constructor.

Comment (byungwook-kim) [reply src/executables/javasp.cpp:214]: please check the windows build fail

**Review by shparkcubrid — APPROVED** [review]

**Review by byungwook-kim — APPROVED** [review]

File: src/executables/javasp.cpp (hunk 1 above)
Comment (hgryoo) [reply src/executables/javasp.cpp:214]: I fixed it, thank you.

**Pull request merged**